operator/src/main/java/oracle/kubernetes/utils/OperatorUtils.java

```diff
@@ -19,4 +25,84 @@ public static String joinListGrammatically(final List<String> list) {
             .concat(list.get(list.size() - 1))
         : list.get(0);
   }
+
+  /**
+   * Create a Map using the elements from the given map, with their keys sorted
+   * using the 'numero lexi sorting name'.
```

Comment:
might be nice to include a link to an explanation of this - i googled that phrase and found nothing that seemed related

Reply:
See pull https://github.com/oracle/weblogic-kubernetes-operator/pull/1503

Reply:
It's a made up term.  Feel free to come up with a better one.

Reply:
https://jira.oraclecorp.com/jira/browse/OWLS-80943

Reply:
there is some description in the comment of getSortingString method in the same class